converter.py

```diff
@@ -0,0 +1,43 @@
+import csv
+import exceptions
+import factory
+from chardet.universaldetector import UniversalDetector
+from models.arshin import Application
+from xsdata.formats.dataclass.serializers import XmlSerializer
+
+
+def detect_encoding(file):
```

Comment:
Please add type hints for all function/method arguments, return values and class properties if they're present.